webpack/assets/javascripts/react_app/components/common/forms/NumericInput.scss

```diff
@@ -0,0 +1,95 @@
+@import "~@theforeman/vendor/scss/variables";
+
+.foreman-numeric-input {
+  position: relative;
+  display: flex;
+
+  .foreman-numeric-input-input-wrap {
+    width: 100%;
+
+    input {
+      flex: 1;
+      height: 26px;
+      width: 100%;
+      padding-right: $padding-small-vertical;
+      box-sizing: border-box;
+      font-size: inherit;
+      border: 1px solid $input-border;
+      border-radius: $input-border-radius;
+      padding-left: $padding-small-vertical;
+      display: block;
+      appearance: none;
+      line-height: normal;
+
+      &:focus {
+        outline: $input-border-focus auto 0.5px;
+      }
+
+      &:hover {
+        border-color: $input-border-hover;
```

Comment:
When I set the disabled prop to true, the hover effect still exists and it looks odd.